configs/container.config

```diff
@@ -3,6 +3,6 @@ process {
     withLabel: filtlong     { container = 'nanozoo/filtlong:0.2.0--0c4cbe3'}
     withLabel: rasusa       { container = 'staphb/rasusa:2.1.0'}
     withLabel: flye         { container = 'staphb/flye:2.9.5'}
-    withLabel: medaka       { container = 'rkimf1/medaka:2.0.1--ef28b94'}
-    withLabel: dorado       { container = 'nanozoo/dorado:0.9.1--871d1b1'}
+    withLabel: medaka       { container = 'ontresearch/medaka:shac4e11bfa4e65668b28739ba32edc3af12baf7574'}
+    withLabel: dorado       { container = 'ontresearch/dorado:shae9327ad17e023b76e4d27cf287b6b9d3a271092b'}
```

Comment:
use now our Dorado container? 

docker pull rkimf1/dorado:1.1.1--9bbaa37

btw, Christian also tested it and also works on his end.